web-app/src/components/common_styled/InputText.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export default styled.input`
+
+ font-family: inherit;
+ font-size: 100%;
+ line-height: 1.15;
+ margin: 10px 0px 10px 0px;
+ height: 34px;
+ width: 100%;
+
+`;
```

Comment:
need a new line at the bottom here - @William-Unosquare should be able to help you setting up linting rules in IDE